### ISSUE TITLE
Timeout on deploy

### ISSUE THREAD

**Issue by nettum** [beyondcode/forge-cli#9]
Hi! This tool looks awesome!
I just installed it and are now testing on our stage server on Digital Ocean.

After installing the cli tool globally with composer I did the following:
```
▶ forge login
```

```
▶ forge init
```
Chose the correct server and site in forge init.
Then manually edit forge.yml to change env from "production" to "stage".

```
▶ forge info stage
+-----------+-------------------------+
| Key       | Value                   |
+-----------+-------------------------+
| Server    | example-stage           |
| IP        | x.x.x.x                 |
| Site      | example.com             |
| Directory | /public                 |
+-----------+-------------------------+
```

```
▶ forge deploy stage
Deploying site on stage...

In TimeoutException.php line 22:
                                                                                                                                        
  Argument 1 passed to Laravel\Forge\Exceptions\TimeoutException::__construct() must be of the type array, bool given, called in phar:  
  ///example.com/.composer/vendor/beyondcode/forge-cli/builds/forge/vendor/laravel/forge-sdk/src/MakesHttpRequests.php on line 138 
```

Things to note:
1) I logged in with a user that have been added to an circle and for that reason only have access to this server.
2) I manually edited the key in forge.yml from "production" to "stage" as I want this to be our stage server.
3) The deploymentscript itself that we have created on forge (and was copied to deployment inside forge.yml on init) seems to work. The site did actually do a successful deployment, but without the deployment output in the CLI. We do not use quick-deploy. That beeing said we have modifed the standard deploymentscript pretty heavily to get a no downtime deployment with forge.

Not sure why I got the timeout exception. Any suggestions or logs I can check? For the record the timeout seemed to come pretty quickly (like after 10-15 seconds, not 30s or 60s).

System:
```
▶ php -v && forge --version && lsb_release -a
PHP 7.4.10 (cli) (built: Sep  9 2020 06:36:14) ( NTS )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
    with Zend OPcache v7.4.10, Copyright (c), by Zend Technologies
Forge 1.0.5
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.5 LTS
Release:	18.04
Codename:	bionic
```

Thanks!

**Comment by mpociot** [comment]
I just fixed this in version 1.1.0

The timeout was increased, but you can also use the new `--no-wait` option on the deploy command, in case you do not need to wait for the deployment log/result.

**Comment by nettum** [comment]
Thanks! Just tested and it's working smoothly now :+1:
